src/bot/utils/dmmessage/dmmessage.service.ts

```diff
@@ -65,7 +65,7 @@ export class DmmessageService {
           return this.syncRole.execute(message, args, client);
         case "*call":
           return this.requestVoiceCallCommand.execute(message, args, client);
-        case "*where":
+        case "*reply":
```

Comment:
what is this?

Reply:
cái này anh Hải bảo chuyển từ where sang reply ạ